Fix #includes, it appears those files move over time with opencore-amr releases.

diff -Nur a/ext/amrnb/amrnbdec.h b/ext/amrnb/amrnbdec.h
--- a/ext/amrnb/amrnbdec.h	2011-12-30 18:40:45.000000000 +0100
+++ b/ext/amrnb/amrnbdec.h	2013-02-03 17:24:52.099845338 +0100
@@ -22,7 +22,7 @@
 
 #include <gst/gst.h>
 #include <gst/audio/gstaudiodecoder.h>
-#include <interf_dec.h>
+#include <opencore-amrnb/interf_dec.h>
 
 G_BEGIN_DECLS
 
diff -Nur a/ext/amrnb/amrnbenc.h b/ext/amrnb/amrnbenc.h
--- a/ext/amrnb/amrnbenc.h	2011-12-30 18:40:45.000000000 +0100
+++ b/ext/amrnb/amrnbenc.h	2013-02-03 17:25:02.554738906 +0100
@@ -21,8 +21,8 @@
 #define __GST_AMRNBENC_H__
 
 #include <gst/gst.h>
-#include <interf_enc.h>
 #include <gst/audio/gstaudioencoder.h>
+#include <opencore-amrnb/interf_enc.h>
 
 G_BEGIN_DECLS
 
diff -Nur a/ext/amrwbdec/amrwbdec.h b/ext/amrwbdec/amrwbdec.h
--- a/ext/amrwbdec/amrwbdec.h	2011-12-30 18:40:45.000000000 +0100
+++ b/ext/amrwbdec/amrwbdec.h	2013-02-03 17:26:07.553077295 +0100
@@ -22,8 +22,8 @@
 
 #include <gst/gst.h>
 #include <gst/audio/gstaudiodecoder.h>
-#include <dec_if.h>
-#include <if_rom.h>
+#include <opencore-amrwb/dec_if.h>
+#include <opencore-amrwb/if_rom.h>
 
 G_BEGIN_DECLS
 
